cs-powershell/src/main/java/io/cloudslang/content/utils/WSManUtils.java

```diff
@@ -1,13 +1,33 @@
 package io.cloudslang.content.utils;
 
+import io.cloudslang.content.entities.EncoderDecoder;
+import io.cloudslang.content.entities.WSManRequestInputs;
+import org.apache.commons.io.Charsets;
 import org.apache.commons.lang3.StringUtils;
 import org.xml.sax.SAXException;
 
 import javax.xml.parsers.ParserConfigurationException;
 import javax.xml.xpath.XPathExpressionException;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
 import java.util.UUID;
 
+import static io.cloudslang.content.constants.OtherValues.COMMA_DELIMITER;
+import static io.cloudslang.content.utils.Constants.Others.ENCODED_COMMAND_PARAMETER;
+import static io.cloudslang.content.utils.Constants.Others.IMPORT_MODULE_PARAMETER;
```

Comment:
`imports  ...*`? :laughing:

Reply:
Alright already! You and your stars! Get a room!